### PR TITLE
SensorOverlay - alert color

### PR DESCRIPTION
**Pull request by xitman78**
Show draggable circle with special color if the sensor's value is out of min-max limit.

### CODE REVIEW

File: src/components/SensorOverlay/components/DraggableBox.tsx [hunk 1]
```diff
@@ -396,6 +394,7 @@ export class DraggableBox extends Component<
             <div
               style={{
                 ...handleStyles,
+                backgroundColor: this.isValid() ? 'white' : '#e74c3c',
```
Comment (mrtraser) [inline src/components/SensorOverlay/components/DraggableBox.tsx]: Could we make it (color) configurable?

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-205.gearboxjs.preview.cogniteapp.com

File: src/components/SensorOverlay/components/DraggableBox.tsx (hunk 1 above)
Comment (xitman78) [reply src/components/SensorOverlay/components/DraggableBox.tsx]: yes, I'll update the PR.

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-205.gearboxjs.preview.cogniteapp.com

**Pull request merged**